Final blog post for request show redesign rollout (#508)
openSUSE/obs-landing: _posts/development/2025-09-16-details_new_request-workflow.md

---
layout: post
title: "The New Request Workflow in Detail"
category: development
---

After a couple of months of beta testing, the new request workflow user interface is enabled for everyone now.

> Let's take a closer look at the key improvements and what they mean for you.

Reviewer: ```suggestion
Let's take a closer look at the key improvements and the reasons we have introduced them.
```
